_posts/2021/12/2021-12-09-sc21-summary.md

```diff
@@ -0,0 +1,63 @@
+## The BoF at SC21
+
+The HPC Carpentry team hosted a "Birds of a Feather" 
+session at the [SC21][sc21] conference.
+
+The primary purpose of this was to try to reconnect with 
+the HPC user community, and gather feedback and guidance
+on how to prioritize our efforts moving forward.
+
+We imagine that the SC21 audience is probably somewhat more
+HPC-aware than the Carpentries community at large, and 
+possibly also biased towards facility operators, rather 
+than novice users, so the feedback we gained, while obviously
+valuable, nevertheless does have this basic bias baked
+into it.
+
+This was a hybrid conference, and the BoF sessions were
+done via the Zoom teleconferencing tool, with Q&A done
+via the Sli.do tool.  The SC21 organizers provided access to
+all of these venues for registered attendees.
+
+Andrew Reid was on-site and hosted the on-site portion from 
+the podium, while Trevor Keller, Anajian Alim Rasel, and 
+Alan O'Cais were on-line, monitoring the Zoom and Sli.do 
```

Comment:
```suggestion
the podium, while Trevor Keller, Annajiat Alim Rasel, 
Alan O'Cais, and Wirawan Purwanto were on-line,
monitoring the Zoom and Sli.do 
```

Reply:
Left off Wirawan because he was not listed as a co-host, but you're right that he was in there doing stuff, which also counts.